Iterable/iterable-web-sdk
# Create test (#250)

## JIRA Ticket

* [MOB-7534](https://iterable.atlassian.net/browse/MOB-7534)

## Description

- **Release Notes**
> The required changes for correct email/userId field placement have been completed

> We've also updated the sample app to showcase the changes.

- **Test Steps**
> Start by launching the sample app, then select the 'embedded' option. Enter your userId/email in the designated field. Afterward, choose the option you wish to test and click submit button. You will then be able to observe the parameter changes in the API request.

[MOB-7534]: https://iterable.atlassian.net/browse/MOB-7534?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
